cmd/lncli/commands.go

```diff
@@ -331,16 +332,26 @@ var sendCoinsCommand = cli.Command{
 				"scripts",
 		},
 		coinSelectionStrategyFlag,
+		cli.StringSliceFlag{
+			Name: "utxo",
+			Usage: "a utxo specified as outpoint(tx:idx) which " +
+				"will be used as input for the transaction " +
```

Comment:
nit: sounds like there should be a full stop after "transaction". Or, just add a semicolon and continue lowercase to make it look like one sentence.